### PR TITLE
AverageReview Score methods are added

### PR DESCRIPTION
**Pull request by Niyathi3011**

### CODE REVIEW

File: app/src/main/java/movierating/services/AverageReviewScoreByYear.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package movierating.services;
+
+import movierating.models.Movie;
+import movierating.repositories.MovieRepository;
+
+public class AverageReviewScoreByYear {
```
Comment (shashiCoderDattebayo) [inline app/src/main/java/movierating/services/AverageReviewScoreByYear.java:6]: I dont see any relation between name `AverageReviewScoreByYear` and what it does. There is no year check.

File: app/src/main/java/movierating/services/AverageReviewScoreForMovie.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public class AverageReviewScoreForMovie {
+
+    private final MovieRepository movieRepository;
+
+    public AverageReviewScoreForMovie(MovieRepository movieRepository) {
+        this.movieRepository = movieRepository;
+    }
+
+    public int getAverageReviewScoreByYear(int releaseYear){
```
Comment (shashiCoderDattebayo) [inline app/src/main/java/movierating/services/AverageReviewScoreForMovie.java]: Why is average a int? Shouldn't it be 3.5?

File: app/src/main/java/movierating/services/AverageReviewScoreForMovie.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package movierating.services;
+
+import movierating.models.Movie;
+import movierating.repositories.MovieRepository;
+
+public class AverageReviewScoreForMovie {
```
Comment (shashiCoderDattebayo) [inline app/src/main/java/movierating/services/AverageReviewScoreForMovie.java:6]: Aren't there any test cases to test these in the main?

File: app/src/main/java/movierating/services/AverageReviewScoreByYear.java (hunk 1 above)
Comment (Niyathi3011) [reply app/src/main/java/movierating/services/AverageReviewScoreByYear.java:6]: What dude? Didn't get you.

File: app/src/main/java/movierating/services/AverageReviewScoreForMovie.java (hunk 2 above)
Comment (Niyathi3011) [reply app/src/main/java/movierating/services/AverageReviewScoreForMovie.java]: Ya. Fixed this.

File: app/src/main/java/movierating/services/AverageReviewScoreForMovie.java (hunk 3 above)
Comment (Niyathi3011) [reply app/src/main/java/movierating/services/AverageReviewScoreForMovie.java:6]: Have added few test cases.

File: app/src/main/java/movierating/services/AverageReviewScoreByYear.java (hunk 1 above)
Comment (shashiCoderDattebayo) [reply app/src/main/java/movierating/services/AverageReviewScoreByYear.java:6]: Its done.

**Review by shashiCoderDattebayo — APPROVED** [review]

**Pull request merged**